src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/state/attributesAccordionReducer.ts

```diff
@@ -123,9 +92,9 @@ function Orthologue(
 }
 
 function OrthologueAttributes(
-  state: any = defaultAttributesAccordionState.attributes,
+  state: any = defaultAttributesAccordionState.content,
```

Comment:
defaultAttributesAccordionState.content now has a type that is not `any`

Reply:
Updated it